src/components/views/elements/PollCreateDialog.tsx

```diff
@@ -129,20 +131,26 @@ export default class PollCreateDialog extends ScrollableBaseModal<IProps, IState
                         placeholder={_t("Write an option")}
                         onChange={e => this.onOptionChange(i, e)}
                         usePlaceholderAsHint={true}
+                        disabled={this.state.busy}
                     />
                     <AccessibleButton
                         onClick={() => this.onOptionRemove(i)}
                         className="mx_PollCreateDialog_removeOption"
+                        disabled={this.state.busy}
                     />
                 </div>)
             }
             <AccessibleButton
                 onClick={this.onOptionAdd}
-                disabled={this.state.options.length >= MAX_OPTIONS}
+                disabled={this.state.busy || this.state.options.length >= MAX_OPTIONS}
                 kind="secondary"
                 className="mx_PollCreateDialog_addOption"
                 inputRef={this.addOptionRef}
             >{ _t("Add option") }</AccessibleButton>
+            { this.state.busy ?
+                <div className="mx_PollCreateDialog_busy"><Spinner /></div> :
+                null
+            }
```

Comment:
```suggestion
            { this.state.busy && <div className="mx_PollCreateDialog_busy"><Spinner /></div> }
```

I believe this should work

Reply:
Good idea, thanks - done (using my own commit because I didn't see you'd made this as a suggestion).